src/components/BottomBarWithIcons/index.js

```diff
@@ -66,6 +66,33 @@ function BottomBarWithIcons({ icons, saveIconArray }) {
     }
   }
 
+  const handleSaveIcon = (onClick) => {
+    if (basketHistory) {
+      return
+    } else {
```

Comment:
tu bym nie dawał else, tj. zrobił
if(basketHistory) {
return
}

if (basketEmpty) {
        history.push('/basket/messageWhenEmpty')
      }
...

I tak return zwraca i kończy funkcję, więc to co po ifie się już nie wykona. Więc else jest niepotrzebny, a zagnieżdżenie mniej i czytelniej. Ten else w if na basketEmpty już ma sens

Reply:
Rzeczywiście kod jest bardziej przejrzysty poprawiłem analogicznie następnego if-a.
Poprawione.